checker/bin/wpi.sh

```diff
@@ -19,6 +19,8 @@ while getopts "d:t:b:g:" opt; do
        ;;
     g) GRADLECACHEDIR="$OPTARG"
        ;;
+    c) BUILD_TARGET="$OPTARG"
```

Comment:
"c" is not a great option name, but "b" and "t" are already taken, so OK.

Reply:
I agree! I considered some other names, but didn't like any of them. And I don't want to change any of the existing options, for backwards-compatibility.